### ISSUE TITLE
Modify config for headless server?

### ISSUE THREAD

**Issue by setzke** [Tschipcraft/spawnanimations#20]
### Is your suggested enhancement related to a problem? Please describe.

I have this installed in MOD form (not against changing to Datapack... unaware of pros/cons) for 1.20.2 fabric server, which runs without a client. What can I do to modify the activation distance? I could not find the config file, and the commands don't work from the command line. Running the trigger from a player does not get me into the SETTINGS part. (I'm unfamiliar with functions to troubleshoot it myself).  Please advise!

![2023-12-16_23 33 09](https://github.com/Tschipcraft/spawnanimations/assets/13110348/a1c31f8c-8116-47f6-a47b-ef6bda845c14)

### Describe the solution you'd like

Better documentation in the wiki on manual customizations, or the datapack/mod difference if this requres the datapack version.

**Comment by setzke** [comment]
Figured it out.

With player on:
`/scoreboard players set @a ts.sa.distance 32`
(Changes for everyone who's online, the distance to 32.)
![2023-12-17_00 24 33](https://github.com/Tschipcraft/spawnanimations/assets/13110348/402d1370-030b-4afe-af03-4722afc99cfe)

**Comment by Tschipcraft** [comment]
Thanks for bringing this to my attention, thinking about it the documentation could be improved upon quite a bit..
The problem you were facing was that you didn't have permission to execute commands, right?

**Comment by setzke** [comment]
> Thanks for bringing this to my attention, thinking about it the documentation could be improved upon quite a bit.. The problem you were facing was that you didn't have permission to execute commands, right?

Yeah I learned later on the easiest way was to just use the console to OP my player then I can do whatever.  So that's a non-issue, I guess!  But I did find myself scowering the wiki at first because I was of the mindset I could alter everything via commands directly.  So unless you have a reason not to, I think it wouldn't hurt to throw the scoreboard commands of what controls what into the wiki.

**Comment by Tschipcraft** [comment]
Yes, it's definitely possible, I will add that to the wiki in the future.
